nano/core_test/active_transactions.cpp

```diff
@@ -11,60 +11,62 @@ using namespace std::chrono_literals;
 
 namespace nano
 {
-// Test disabled because it's failing intermittently.
-// PR in which it got disabled: https://github.com/nanocurrency/nano-node/pull/3512
-// Issue for investigating it: https://github.com/nanocurrency/nano-node/issues/3522
-TEST (active_transactions, DISABLED_confirm_active)
+TEST (active_transactions, confirm_active)
 {
-	nano::system system;
-	nano::node_flags node_flags;
-	node_flags.disable_request_loop = true;
-	auto & node1 = *system.add_node (node_flags);
-	auto send = nano::send_block_builder ()
-				.previous (nano::dev::genesis->hash ())
-				.destination (nano::public_key ())
-				.balance (nano::dev::constants.genesis_amount - 100)
-				.sign (nano::dev::genesis_key.prv, nano::dev::genesis_key.pub)
-				.work (*system.work.generate (nano::dev::genesis->hash ()))
-				.build_shared ();
-	ASSERT_EQ (nano::process_result::progress, node1.process (*send).code);
-	nano::node_config node_config2 (nano::get_available_port (), system.logging);
-	node_config2.frontiers_confirmation = nano::frontiers_confirmation_mode::disabled;
-	nano::node_flags node_flags2;
-	// The rep crawler would otherwise request confirmations in order to find representatives
-	node_flags2.disable_rep_crawler = true;
-	auto & node2 = *system.add_node (node_config2, node_flags2);
+	nano::system system{};
 	system.deadline_set (5s);
-	// Let node2 know about the block
-	while (node2.active.empty ())
-	{
-		node1.network.flood_block (send, nano::buffer_drop_policy::no_limiter_drop);
-		ASSERT_NO_ERROR (system.poll ());
-	}
-	// Save election to check request count afterwards
```

Comment:
We are losing this comment and the past two ones. Could you document the new code?

Reply:
These lines specifically should need no comments at all, they're a standard pattern we're using to check that the block has been processed and election started. But yeah, I'll do add some docs here and there where they would help more.